nether/init.lua

```diff
@@ -52,6 +52,9 @@ nether.start = f_h_max+100
 -- Height of the nether (bottom of the nether is nether_middle - NETHER_HEIGHT)
 local NETHER_HEIGHT = 30
 
+-- bottom height of the nether.
+nether.bottom = nether_middle - NETHER_HEIGHT
```

Comment:
>  Testing shows that nether generation goes lower than y=-20030. The bottom of the nether on this map is at y=-20190. What am I missing?

The nether mapgen code currently does not use a clean way to determine the bottom of the nether, so it depends on the mapchunk size, which is usually 80 nodes and can be changed with the [`chunksize` setting](https://github.com/minetest/minetest/blob/fbec378869691444e354425073a7210555a9b31d/minetest.conf.example#L3047-L3054).

With 80 nodes mapchunk size, the bottom could be between `NETHER_BOTTOM-179` and `NETHER_BOTTOM-116` or something like that, so I think `nether_middle - NETHER_HEIGHT - 100` could fit a bit closer to the actual bottom.

Reply:
Thanks.